_includes/news-list-blocks.html

```diff
@@ -3,13 +3,16 @@
     <h3>Latest News</h3>
   </div>
   {% for post in site.posts %}
-    {% assign author = site.data.authors[post.author] %}
+    {% assign author = "" | split: ',' %}
```

Comment:
Is there a reason for this `split: ','` part?

Reply:
it creates an empty array in Liquid, the language doesn't seem to have a native way of doing it